app/subcommands/service_subcommand.py

```diff
@@ -54,3 +59,14 @@ def remove_pid_file():
             except OSError:
                 pass
         UnhandledExceptionHandler.singleton().add_teardown_callback(remove_pid_file)
+
+    def _get_https_options(self):
+        https_cert_file = Configuration['https_cert_file'] if 'https_cert_file' in Configuration else None
+        https_key_file = Configuration['https_key_file'] if 'https_key_file' in Configuration else None
```

Comment:
Minor:

I think you could probably just remove these if 'key' in configuration checks, because you always default it to None.